src/dataset.py

```diff
@@ -39,10 +40,10 @@ def license_lookup(license: str) -> str:
 
 def name_lookup(
     metadata: api_types.DatasetMetadata,
-    entities: list[api_types.Entity],
+    entities: typing.List[api_types.Entity],
     strict_search: bool = None,
     **kwargs,
-) -> str:
+) -> typing.Tuple[str, bool]:
```

Comment:
Not important now, but just as a note for the future - I typically do `from typing import <stuff>` for the typing library to make it easier to type out :smile: